src/schema/feeds.ts

```diff
@@ -923,6 +963,21 @@ export const resolvers: IResolvers<any, Context> = traceResolvers({
       }
       return feedResolverV1(source, args, ctx, info);
     },
+    feedPreview: (source, args: ConfiguredFeedArgs, ctx: Context, info) => {
+      if (process.env.NODE_ENV === 'development') {
+        return feedResolverV1(source, args, ctx, info);
+      }
```

Comment:
Not sure how else to make `feedResolverV2` work locally other then this, in `feed` query we send `version: 1` from frontend locally but here there is no version really.

Reply:
can you elaborate on why this is needed? Feels like we should not have such high divergence in code between dev & prod

Reply:
Cant you just use `feedByConfig` as the request method?
(or another existing feed request)

Reply:
Because `feedResolverV2` does not work locally. I agree with divergence that is why I wrote the comment 🤝 . For me true resolving of this issue is to make `feedResolverV2` work locally the same way `feedResolverV1` works.

Reply:
Ah let's not introduce things just for a local env fix then.
Worst case we can mimic it locally temporary, or indeed as you said make v2 work/mimic on local envs

Ideally we actually get rid of v1 as it's only used locally.

Reply:
Removed it in here 6373e93497ce85df87df73e055fdee5f169efa02 but it is not gonna work locally now. I think we need to find a way to mock resolver v2 locally in near time.

Reply:
Maybe for the time beeing use something like Mockooon or something like that?
We need to look into spooling it up as a service separate to this ticket, we also need it for auth soon.

Reply:
Yeah, don't have the preference, whatever is easiest. Is there a task already for that or should I open it?